core/modules/views_ui/views_ui.admin.inc

```diff
@@ -3775,86 +3775,87 @@ function views_ui_add_item_form($form, &$form_state) {
   // Add the display override dropdown.
   views_ui_standard_display_dropdown($form, $form_state, $section);
 
-  // Figure out all the base tables allowed based upon what the relationships provide.
+  // Figure out base tables allowed based upon what the relationships provide.
   $base_tables = $view->get_base_tables();
-  $options = views_fetch_fields(array_keys($base_tables), $type, $display->handler->use_group_by());
+  $fields = views_fetch_fields(array_keys($base_tables), $type, $display->handler->use_group_by());
 
-  if (!empty($options)) {
-    $form['options']['controls'] = array(
-      '#theme_wrappers' => array('container'),
-      '#id' => 'views-filterable-options-controls',
-      '#attributes' => array('class' => array('container-inline')),
-    );
-    $form['options']['controls']['options_search'] = array(
-      '#type' => 'textfield',
-      '#title' => t('Search'),
-    );
-
-    $groups = array('all' => t('- All -'));
-    $form['options']['controls']['group'] = array(
-      '#type' => 'select',
-      '#title' => t('Filter'),
-      '#options' => array(),
-    );
+  $form['options']['controls'] = array(
+    '#theme_wrappers' => array('container'),
+    '#id' => 'views-filterable-options-controls',
+    '#attributes' => array('class' => array('container-inline')),
+  );
+  $form['options']['controls']['options_search'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Search'),
+  );
 
-    $form['options']['name'] = array(
-      '#prefix' => '<div class="views-radio-box form-checkboxes views-filterable-options">',
-      '#suffix' => '</div>',
-      '#tree' => TRUE,
-      '#default_value' => 'all',
-    );
+  $groups = array('all' => t('- All -'));
```

Comment:
Should we be using [`'#empty_option'`](https://docs.backdropcms.org/form_api#empty_option) here instead?

Reply:
Thanks @klonos, please see if the last commit addresses your concerns:
https://github.com/backdrop/backdrop/pull/4628/commits/6ac48cf6903e24022e5830797f270016e0b31a91

Reply:
Yes, it does 👍🏼 

Thanks